cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -239,7 +249,18 @@ def endpoint_dial(self, endpoint_str, timeout=3):
         for prefix in protocols:
             try:
                 ep = "{}://{}".format(prefix, endpoint_str)
-                r = requests.head(ep, timeout=timeout)
+                # If verify is set to a path to a directory,
+                # the directory must have been processed using the c_rehash utility supplied with OpenSSL.
+                if prefix == "https" and cert and self._arg_parser.rgw_skip_tls:
+                    r = requests.head(ep, timeout=timeout, verify=False)
+                if prefix == "https" and cert:
+                    r = requests.head(ep, timeout=timeout, verify=cert)
+                # TLS without cert?
+                elif prefix == "https":
```

Comment:
IMO it is better to fail the endpoint check if the user gives HTTPS without a cert, if it does not fail here, it will fail in `healthbuckerchecker`, `OBC` and `cephobjectstoreuser` operations.